tracing/jaeger/src/main/java/io/helidon/tracing/jaeger/JaegerTracerBuilder.java

```diff
@@ -299,6 +306,11 @@ public JaegerTracerBuilder config(Config config) {
 
         config.get("global").asBoolean().ifPresent(this::registerGlobal);
 
+        config.get("exporter-timeout-millis").asLong().ifPresent(it -> exporterTimeout(Duration.ofMillis(it)));
```

Comment:
switch to Duration instead of `millis` - so the property should be `exporter-timeout` (same for schedule delay)

Reply:
fixed